docs/source/flow-project.rst

```diff
@@ -39,44 +39,103 @@ Executing this script on the command line will give us access to this project's
 
 .. note::
 
-    You can have **multiple** implementations of :py:class:`~flow.FlowProject` that all operate on the same **signac** project!
+    You can have **multiple** implementations of :py:class:`~flow.FlowProject` that all operate on the same **signac** data space!
     This may be useful, for example, if you want to implement two very distinct workflows that operate on the same data space.
     Simply put those in different modules, *e.g.*, ``project_a.py`` and ``project_b.py``.
 
-.. _workflow-definition:
```

Comment:
Don't have time to check rn if this was used before/if it needs to be added to a redirect list or anything for any people's saved URLs.

Reply:
Currently the url that links to the Defining a workflow section. 

https://docs.signac.io/en/latest/flow-project.html#defining-a-workflow

Perhaps the right course of action is to leave both targets for this section?